integration_test/agent_metrics/metadata.yaml

```diff
@@ -0,0 +1,237 @@
+expected_metrics:
+- type: agent.googleapis.com/agent/api_request_count
+  value_type: INT64
+  kind: CUMULATIVE
+  monitored_resource: gce_instance
+  labels:
+    state: .*
+- type: agent.googleapis.com/agent/log_entry_count
+  value_type: INT64
+  kind: CUMULATIVE
+  monitored_resource: gce_instance
+#  TODO(b/205585572#comment2) Add response_code when it becomes available
+#  labels:
+#    response_code: ^[0-9]+$
+#  TODO(b/170138116): Enable this metric once it is being collected.
+#- type: agent.googleapis.com/agent/log_entry_retry_count
+#  value_type: INT64
+#  kind: CUMULATIVE
+#  monitored_resource: gce_instance
+#  labels:
+#    response_code: .*
+#  optional: true
+- type: agent.googleapis.com/agent/memory_usage
+  value_type: DOUBLE
+  kind: GAUGE
+  monitored_resource: gce_instance
+- type: agent.googleapis.com/agent/monitoring/point_count
+  value_type: INT64
+  kind: CUMULATIVE
+  monitored_resource: gce_instance
+  labels:
+    status: .*
+- type: agent.googleapis.com/agent/request_count
+  value_type: INT64
+  kind: CUMULATIVE
+  monitored_resource: gce_instance
+#  TODO(b/205585572#comment2) Add response_code when it becomes available
+#  labels:
+#    response_code: ^[0-9]+$
+- type: agent.googleapis.com/agent/uptime
+  value_type: INT64
+  kind: CUMULATIVE
+  monitored_resource: gce_instance
+  labels:
+    version: .*
+  representative: true
+- type: agent.googleapis.com/cpu/load_15m
+  value_type: DOUBLE
+  kind: GAUGE
+  monitored_resource: gce_instance
+- type: agent.googleapis.com/cpu/load_1m
+  value_type: DOUBLE
+  kind: GAUGE
+  monitored_resource: gce_instance
+- type: agent.googleapis.com/cpu/load_5m
+  value_type: DOUBLE
+  kind: GAUGE
+  monitored_resource: gce_instance
+- type: agent.googleapis.com/cpu/utilization
```

Comment:
`agent.googleapis.com/cpu/usage_time` is documented for both ops-agent and legacy agent.

Although https://screenshot.googleplex.com/56R46mk7ECNbk2W shows we never had such metric descriptors before, shall we check if any client reports this metric, if not, shall we remove it from the public doc @qingling128 @igorpeshansky

Reply:
Added in: [cf8f612](https://github.com/GoogleCloudPlatform/ops-agent/pull/719/commits/cf8f61234082bc521e24290d1f64d70b9a784454)